### ISSUE TITLE
Uncaught Error: An element for 'Container6umlajn' could not be found in [...]

### ISSUE THREAD

**Issue by tominventisbe** [twifty/atom-phpunit-integrator#56]
I think this happens because I generated code coverage and a subfolder disappeared afterwards. These are folders automatically generated and maintained by Symfony in the `/var/cache/dev` folder.

```
/home/user/.atom/packages/php-unit-integrator/lib/views/php-unit-tree-view.js:375

Uncaught Error: An element for 'Container6umlajn' could not be found in [Container8Cmcv4s,ContainerW8oyjKV,doctrine,pools,profiler,rulerz,twig,annotations.map,annotations.php,appDevDebugProjectContainer.php,appDevDebugProjectContainer.php.meta,appDevDebugProjectContainer.xml,appDevDebugProjectContainer.xml.meta,appDevDebugProjectContainerCompiler.log,appDevDebugProjectContainerDeprecations.log,appDevDebugProjectContainerUrlGenerator.php,appDevDebugProjectContainerUrlGenerator.php.meta,appDevDebugProjectContainerUrlMatcher.php,appDevDebugProjectContainerUrlMatcher.php.meta,Container8Cmcv4s.legacy,templates.php,validation.php]!
```

Thanks!

**Comment by twifty** [comment]
Thanks for reporting.

Code coverage information shouldn't be available for items within 'dev' on a Symfony based project. Though, since Symfony3 does generate code within that directory, all files would be listed in the clover.xml (Unless configured to ignore in phpunit.xml). The same may apply to any files generated as part of a test.

I'll change that error to a console warning. I'll also see about making it more descriptive.

**Comment by twifty** [comment]
Fixed in 0.0.53
